proposals/nnnn-multiple-modules.md

```diff
@@ -0,0 +1,663 @@
+Every time a new module is added, a new proposal needs to be approved, and there will be a version change in mobile API.
+
+In this option, the module ID and the module type together identify a module.
+
+There is a "default", “main" or "primary" module ID for each module type. This default module ID may be used if there is only one module per module type. This ID may be also used when there are multiple modules but one of them is the most commonly used.
+
+There is a "driver" module ID. It is potential for the modules that are designed or mostly used for a driver. For example, driver climate, driver seat, driver light, driver audio.
+
+There is a "passenger" or "passengers" module ID. To tell which passenger from all possible passengers, it may have "front passenger" or "rear row passengers". To extend the idea, the name of a location or seat position may be introduced. For example, rear left (the 2nd row vehicle), rear right (2nd row vehicle), second row, third row, etc. It is a short version of "grid" system with only commonly used location defined.
+
+Consider a normal passenger vehicle with 2~8 seats. There are one, two or three rows: driver/front row only; front row and rear row; or front-middle-back rows; or equivalently front row, second row, and third row. There are usually two or three columns in a row: driver-passenger sides, left-right sides or left-middle-right sides.
+
+For vehicles with more (9, 12, 15) seats, there may be four or five rows and four seats per row. It get complex to give a name to each seat location. The "grid" system will work better. However, it is not common to have a RC module specific to a seat in these vehicles.
+
+We listed the most commonly use name here. 
+
+```xml
+<enum name="ModuleId">
+    <description>most commonly used names or locations for a RC module. </description>
+    <element name="DEFAULT"/> <!--main, default, front middle, center -->
+    <element name="DRIVER"/>
+    <element name="FRONT_PASSENGER"/>
+    <element name="SECOND_ROW"/> <!--second row middle or as a whole row -->
+    <element name="SECOND_ROW_LEFT"/>
+    <element name="SECOND_ROW_RIGHT"/>
+    <element name="THIRD_ROW"/> <!-- third row middle or as a whole row -->
+    <element name="THIRD_ROW_LEFT"/>
+    <element name="THIRD_ROW_RIGHT"/>
+</enum>
+```
+
+Not all combinations are valid. OEM shall make sure they publish a valid combination of `ModuleId` for each `moduleType` in `XyzControlCapabilities`. In addition, mobile apps shall only use IDs that are published by the control capabilities.
+
+#### Option 4: SDLC assigned "module" ID
+Similar to option 3, but module ID include module type info. It is possible to use module ID alone to identify a module. We listed the most commonly use name as below. 
+
+```xml
+<enum name="ModuleId">
+    <description>most commonly used RC modules</description>
```

Comment:
I believe all of these descriptions should ready "...used when supporting only one..."

Reply:
updated